scripts/zones/Windurst_Walls/npcs/Shantotto.lua

```diff
@@ -57,6 +57,16 @@ function onTrigger(player,npc)
     elseif (player:getQuestStatus(WINDURST,dsp.quest.id.windurst.CLASS_REUNION) == QUEST_ACCEPTED and player:getVar("ClassReunionProgress") == 3) then
         player:startEvent(409) -- she mentions that Sunny-Pabonny left for San d'Oria
     -------------------------------------------------------
+
+    -- AMK
+    elseif player:getCurrentMission(AMK) == dsp.mission.id.amk.CURSES_A_HORRIFICALLY_HARROWING_HEX then
+        player:startEvent(506)
+    elseif player:getCurrentMission(AMK) == dsp.mission.id.amk.AN_ERRAND_THE_PROFESSORS_PRICE and player:hasKeyItem(dsp.ki.RIPE_STARFRUIT) then
+        player:startEvent(507)
+        -- TODO: Where to check for 5000gil being available?
```

Comment:
Does the event take a param telling it how much gil the player currently has? If the player says yes to the CS, when the CS has started, but they have less than 5,000 gil, what happens?

I have a feeling that it does take a gil value (at least, newer events do).